lib/modules/manager/terragrunt/modules.ts

```diff
@@ -62,7 +62,16 @@ export function analyseTerragruntModule(
     const moduleParts = source.split('//')[0].split('/');
     if (moduleParts[0] === '..') {
       dep.skipReason = 'local';
-    } else if (moduleParts.length >= 3) {
+    } else if (source.startsWith('tfr:///')) {
+      const hostnameMatch = hostnameMatchRegex.exec('registry.terraform.io');
+      if (hostnameMatch?.groups) {
+        dep.registryUrls = [`https://${hostnameMatch.groups.hostname}`];
+      }
```

Comment:
This result will always be the same, so can be refactored/simplified

Reply:
Can you point me to place where I can see test for some of the other part of the code? tried to find it myself didn't find it.